crates/oxc_linter/src/rules/eslint/no_new_wrappers.rs

```diff
@@ -91,5 +100,27 @@ fn test() {
         ",
     ];
 
-    Tester::new(NoNewWrappers::NAME, pass, fail).test_and_snapshot();
+    let fix = vec![
+        ("var a = new String('hello');", "var a = String('hello');"),
```

Comment:
Thank you, I'm sorry, could I refuse this fixer? This changes the code meaning and seems `String('hello')` is not also good:

```js
typeof new String('hello') // 'object'
typeof String("hello")
```

 I'd like `'hello'`, but it will have other edge cases:
 
```js
new Number(1).toString() // original code

1.toString() // SyntaxError
```

It's acceptable and more safe to change one by one manually.

Reply:
> Thank you, I'm sorry, could I refuse this fixer? This changes the code meaning and seems `String('hello')` is not also good:
> 
> ```js
> typeof new String('hello') // 'object'
> typeof String("hello")
> ```
> 
> I'd like `'hello'`, but it will have other edge cases:
> 
> ```js
> new Number(1).toString() // original code
> 
> 1.toString() // SyntaxError
> ```
> 
> It's acceptable and more safe to change one by one manually.

That makes sense.